right_column.md

```diff
@@ -75,29 +75,39 @@ Python.
   [text instructions](https://coderefinery.github.io/installation/git/#configuring-git),
   [video](https://www.youtube.com/watch?v=WdDTp8NeHBs&t=258s).
 
+
+### Host
+- Naoe Tatara
+
 ### Instructors
 
 - Radovan Bast
 - Anne Fouilloux
 - Bjørn Lindi
 - Thor Wikfeldt
 - Stefan Negru
+- Richard Darst
 
 
+### Expert helpers
+
+- Enrico Glerean
+- Jarno Rantaharju
+- Juho Lehtonen
+- Raphaela Heil
+- Sabry Razick
 
 ### Helpers
 
 ([tips for helpers](https://github.com/coderefinery/manuals/blob/master/helping-and-teaching.md))
 
```

Comment:
```suggestion

We have 24 registered helpers who have made this Mega-CodeRefinery possible.  If you would like to be be listed here, let us know.
```

Reply:
It would be good to somehow give credit to helpers so we should point this out during the workshop or in a thank you email.